dockstore-webservice/src/main/java/io/dockstore/webservice/core/Collection.java

```diff
@@ -200,10 +202,15 @@ public void setDescription(String description) {
     }
 
     @JsonProperty("entries")
-    public List<CollectionEntry> getCollectionEntries() {
```

Comment:
This is "entries" as well

Reply:
I took another crack at this and fiddled with the field and property names, in the hope that it was just something simple like "it's ignoring [the first] `entries`, so that behavior carries over to the second `entries`".  Alas, no luck.  Even if you rename the field so there's no name collision, if the `JsonProperty` is "entries", it doesn't appear inthe definition of `Collection` in `openapi.yaml`.  Really weird thing is that the "entries" property _does_ appear in `Category`, which extends `Collection`...

Reply:
`Really weird thing is that the "entries" property does appear in Category, which extends Collection...` 

Yeah, I noticed that too. The "entries" that is appearing is the second one. It could be whatever problem we're having is not inherited, so that's yet another possibility. We could create an abstract Collection class and have a new Collection class.

Reply:
> if the `JsonProperty` is "entries", it doesn't appear inthe definition of `Collection` in `openapi.yaml`

In the worse case scenario, I guess we'd have to rename both. Presumably the UI will be broken whether we rename one or both of them anyway